docs/node-operators/how-to/upgrade-network.md

```diff
@@ -161,11 +175,67 @@ In the Vega core logs, you will see the following messages:
 ...
 ```
 
-#### Check if data node is ready for restart
-To see when your data node is ready for protocol upgrade, check the data node logs.
-
-In the data node logs, you will see the following message:
+`(b)`: In the data node logs, you will see the following message:
 
 ```
 2023-03-02T13:01:04.379+0100	INFO	datanode.service	service/protocol_upgrade.go:36	datanode is ready for protocol upgrade
-```
\ No newline at end of file
+```
+
+## 3.3 Restart services
+
+`(a)`: Restart your `core` process
+`(b)`: When you run both: `core` and `data-node`, you must stop both first, and then you can start both in any order.
+
+## 3.4 Automate
+
+Vega Visor is designed to perform `Protocol Upgrade` automatically, but you can automate it yourself. The key is to check if `core` is ready for a restart for `Protocol Upgrade`. As described before, it might take some time (seconds) between the node reaching the `Protocol Upgrade` block and marking itself as ready for the restart. That is why you should not rely on block height, e.g. from `/statistics`.
+
+Instead, you need to query `core` using the Admin json rpc endpoint. This is what Vega Visor is doing. Important Note: We do not have any plans, but this might change, so please watch for changes in the Vega Visor.
```

Comment:
```suggestion
Instead, you need to query `core` using the Admin json rpc endpoint as this is what Vega Visor is doing. Note: This might change in the future, so please keep an eye on the release notes for changes in the Vega Visor.
```